bsc/0x265f638eAb14dD43D80bB71AAe3289B1c6d46A3f/info.json

```diff
@@ -0,0 +1,12 @@
+{
+ "name": "Binancedog",
+ "website": "https://www.binancedog.net/",
+ "description": "🐶Binance dog 🐶is a deflationary token which is centered by the community and driven by the internal force. 

```

Comment:
"description": "🐶Binance dog 🐶is a deflationary token which is centered by the community and driven by the internal force.",